logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/user/SyncSelfUserUseCase.kt

```diff
@@ -25,11 +25,15 @@ import com.wire.kalium.logic.functional.Either
 /**
  * This use case will sync the current user with the backend.
  */
-class SyncSelfUserUseCase internal constructor(
+internal interface SyncSelfUserUseCase {
+    suspend operator fun invoke(): Either<CoreFailure, Unit>
+}
```

Comment:
Nice catch making these internals.

These are so old that we didn't have this standard back then :D